domains/devon.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Devon Powell is a full-stack software engineer",
+    "repo": "https://github.com/D-Pow/website",
+    "owner": {
+        "username": "D-Pow",
+        "email": "dpow9373@gmail.com"
+    },
+    "record": {
+        "ALIAS": "https://d-pow.github.io/website/",
```

Comment:
We have removed support for alias records. Also, this is an invalid alias/cname.

You should rename your `website` repository to `d-pow.github.io`. You can read more about github pages [here](https://guides.github.com/features/pages/)

```suggestion
        "CNAME": "d-pow.github.io",
```

Reply:
@phenax I have pushed the recommended changes.

Relatedly, I realize this is to redirect from e.g. GitHub pages to the name.is-a.dev URL. May I ask what type of hosting the is-a.dev domain is providing? I've had issues with stealth hosting systems before because they mess up certain JavaScript window-size readings.

Reply:
It is just a subdomain with a cname dns record. Not sure if that would have any effect on the javascript executing on the page.
I'm guessing what you are talking about is some kind of iframed thing which would explaing any viewport issues with the js. In that case, you won't have that problem here as it is just a dns record.

Reply:
Yes, the iframe is exactly what I mean. If that's not the case, then there won't be any issues. Thanks for the details and suggestions above!